assets/js/buttons-examples.js

```diff
@@ -173,6 +173,11 @@ $(document).ready(function () {
       classes: "outline-btn",
       children: [],
     },
+    {
```

Comment:
Place Disabled Buttons in the correct alphabetical position.

Reply:
Hi @shahednasser !! placed it below animated buttons! Please review! thank you!